content/docs/get-started-with-neon/query-with-psql-editor.md

```diff
@@ -1,20 +1,124 @@
 ---
 title: Query with psql Editor
+enableTableOfContents: true
 ---
 
-To follow this guide, you will need a working installation of [psql](https://www.postgresql.org/download/), the PostgreSQL interactive terminal.
+The following steps require a working installation of [psql](https://www.postgresql.org/download/), a terminal-based front-end to PostgreSQL. For information about `psql`, refer to the [psql reference](https://www.postgresql.org/docs/14/app-psql.html), in the _PostgreSQL Documentation_. 
```

Comment:
Hey @danieltprice , I'm not sure about 'a terminal-based front-end' part
Here is a link to psql definition https://www.postgresguide.com/utilities/psql/

Reply:
Fixed. Thank you!